src/__tests__/TabNav.tsx

```diff
@@ -29,19 +29,8 @@ describe('TabNav', () => {
     expect(render(<TabNav />).type).toEqual('nav')
   })
 
-  it('adds the TabNav class', () => {
-    expect(rendersClass(<TabNav />, 'TabNav')).toEqual(true)
-  })
 
   it('sets aria-label appropriately', () => {
     expect(render(<TabNav aria-label="foo" />).props['aria-label']).toEqual('foo')
   })
-
-  it('wraps its children in an "TabNav-body" div', () => {
```

Comment:
These tests were failing because they depended on the class names that I removed, but after thinking through them, they both just test implementation details which felt silly. 

The first tests if there's a class on the element, but that class isn't used anywhere and isn't necessary for any functionality.

The second test is basically testing that it renders children, which is something that we can assume React will do just fine 👍